lib/elixir/lib/function.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Function do
+  @moduledoc """
+  A set of funtions for working with functions.
+
+  We can have two types of captures functions - external and local -
+  external functions are our normal functions residing in modules while
+  local are the anonymous functions defined with `fn` or the capture
+  operator `&`.
```

Comment:
I think that fn and & should have the slash and the arity so it cane be linked in the documentation.. but im not too sure here how we are dealing with it somewhere else